pkg/loghelper/loghelper.go

```diff
@@ -63,3 +66,32 @@ func SetLogFormat() {
 	}
 	logrus.SetFormatter(customFormatter)
 }
+
+func SetHooks(f *os.File) {
+	logrus.AddHook(&writer.Hook{ // Send all logs to file
+		Writer: f,
+		LogLevels: []logrus.Level{
+			logrus.PanicLevel,
+			logrus.FatalLevel,
+			logrus.ErrorLevel,
+			logrus.WarnLevel,
+			logrus.InfoLevel,
+			logrus.DebugLevel,
+			logrus.TraceLevel,
+		},
+	})
+}
+
+// setLogLevel sets the log level for logrus based on the "TNF_LOG_LEVEL" environment variable
+func SetLogLevel() {
+	params := configuration.GetTestParameters()
+
+	var logLevel, err = logrus.ParseLevel(params.LogLevel)
+	if err != nil {
+		logrus.Error("TNF_LOG_LEVEL environment set with an invalid value, defaulting to debug \n Valid values are: trace, debug, info, warn, error, fatal, panic")
+		logLevel = logrus.DebugLevel
```

Comment:
I know this is a copy/paste from the old code, but I have to remark this again, as this might be a good moment to change it: typically, in production software, the default log level for any application is (and should be) "info". Only when something went wrong, the user (probably a developer at this point) wants to see the "debug" output, so he/she will run again the test suite with the debug level set. If the debug output is not enough to fix the issue, we also have the "trace" level which should show everything else.

Reply:
Done, changed to `info`.